syntax = "proto2";

message request_msg0
{
	required string info = 1;
}